.vscode/tasks.json

```diff
@@ -3,13 +3,32 @@
     "tasks": [
         {
             "label": "build",
-            "command": "dotnet",
+            "command": "pwsh",
             "type": "process",
             "args": [
-                "build",
-                "${workspaceFolder}/psworker.csproj"
+                "-c",
+                "${workspaceFolder}/build.ps1"
             ],
-            "problemMatcher": "$msCompile"
+            "problemMatcher": "$msCompile",
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            }
+        },
+        {
+            "label": "test",
+            "command": "pwsh",
+            "type": "process",
+            "args": [
+                "-c",
+                "${workspaceFolder}/build.ps1",
+                "-Test"
+            ],
+            "problemMatcher": "$msCompile",
+            "group": {
+                "kind": "test",
+                "isDefault": true
```

Comment:
Both tasks are default, so will both run when build in vscode? Note that `build.ps1 -Test` will build the project again.

Reply:
The `kind` is different. You can have a default Build task and a default Test task. I think the Test task should still build.

That way you can simply ctrl+shipt+p => "Test" and run "Tasks: Run Test Task"